src/Symfony/Component/JsonEncoder/Tests/Fixtures/Model/DummyWithPhpDoc.php

```diff
@@ -21,7 +21,7 @@ class DummyWithPhpDoc
      */
     public static function castArrayOfDummiesToArrayOfStrings(mixed $arrayOfDummies): mixed
     {
-        return array_column('name', $arrayOfDummies);
+        return array_column($arrayOfDummies, 'name');
```

Comment:
Is this code really called? The arguments were reversed, which produces a TypeError.

Reply:
Indeed, first I thought it was an alternative signature but you're right, it's not called (neither the other method below). I removed them.